### DISCUSSION TITLE
Why are quotes changed to \" on my web server?

### DISCUSSION THREAD

**Discussion by wobintosh** [datenstrom/community#131] [Ask a question] [answered]
If I insert a link with HTML-syntax, yellow will mask the ":

```
<a href="http://www.spiegel.de/">link title</a>
```

is changed to...

```
<a href=\"http://www.spiegel.de/\">link title</a>
```

For any reason, the link is working, but the syntax shouldn't be like this.

**Comment by wunderfeyd** [comment]
Can't reproduce this issue (current Yellow snapshot, PHP5.6.9/Apache), more information is needed.

Can you supply us with the information about:
- Where did you change the text? A normal page or something in a config or php file?
- How did you change it? Did you use the webinterface or a text editor to create the modification?
- Is the wrong masked HTML written to disk or is it created on the fly during the page load?
- Have you tried the latest Yellow snapshot?
- Did you change depending parameters of the config in any way or it is the default one?
- What's your application setup? Like PHP version, web server version and browser version.

**Comment by wobintosh** [comment]
I changed the text of a blogpost, used the webinterface. The wrong masked HTML is also written to disk, in some cases the Link actually doesn't work because Apache believes I want to view http://my.website.de/edit/\"http://spiegel.de\"
My yellow is v0.6.1. Beside of an filter for the feed I only changed Sitename, Author and Language.
Apache 2.5.17 with PHP 5.3.29; Firefox 42.0

**Comment by wunderfeyd** [comment]
Thank you @wobintosh2 for the fast response, we try to dig deeper.

@markseu: Any chance this could be related to bc8b86ffad6c33201178f9238753ad3c7633aa27? I see a change for magic quotes (I think for 5.3 only, see [Magic quotes on php.net](https://secure.php.net/manual/en/security.magicquotes.what.php)). Can you reproduce this on PHP 5.3 with magic quotes disabled/enabled?

**Comment by markseuffert** [comment, marked as answer]
Yup, sound like a problem with PHP 5.3 and magic quotes.

Please download the latest [core plugin](https://github.com/annaesvensson/yellow-core). Does that help?

**Comment by wobintosh** [comment]
Yes, with the latest core.php it works fine. Thank you! :)

**Comment by wunderfeyd** [comment]
We're happy to hear it works for you. Thank you for your detailed report.

Have a nice day/week. You're welcome. :)
